integration/configs.go

```diff
@@ -145,6 +145,7 @@ var (
 		if legacy {
 			return map[string]string{
 				"-api.response-compression-enabled":  "true",
+				"-ruler.rule-path":                   "/data/rules",
```

Comment:
This should be always set in the ruler, not just when the legacy object storage config is used. I would either specify below as well or move it to `NewRuler()` if we don't expect it to change between different configs.

Reply:
Good point, I have added it to `NewRuler()` as suggested